apps/students/views.py

```diff
@@ -173,14 +172,22 @@ def ajax_attendance(request):
     return JsonResponse(data)
 
 
+def ajax_mark_attendance(request):
```

Comment:
Add the two decorators above this function. (Refer to the other `ajax_fetch_students` function above).